app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/ViewerFragment.java

```diff
@@ -59,12 +65,18 @@
         SendGeographicMessageDialog.SendGeographicMessageDialogInterface,
         ShowMessageDialogFragment.ShowMessageDialogFragmentInterface, OnGGMapReadyListener {
 
+    private static final Logger sLogger = LoggerFactory.create(ViewerFragment.class);
+
+    //todo: config. after merging configuration branch move this configuration to Constants.java
```

Comment:
why 5 seconds? definitely too much. maybe one minute? maybe 10?
we will decide when merging to dev (and move to Constants)